pom.xml

```diff
@@ -351,6 +351,15 @@
   </dependency>
 <!-- Elasticsearch end -->
 
+<!-- FiniteStateMachine begin -->
```

Comment:
This doesn't appear to be in the FiniteStateMachineMeta.java...  
Please generate the pom file and make sure all changes are updated in the appropriate places.

Reply:
the metas were updated and regenned in a later pr